fixtures/american-dj/revoscan-led.json

```
@@ -0,0 +1,358 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "RevoScan LED",
+  "categories": ["Barrel Scanner"],
+  "meta": {
+    "authors": ["Daniel D."],
+    "createDate": "2024-09-26",
+    "lastModifyDate": "2024-09-26"
+  },
+  "links": {
+    "video": [
+      "https://www.youtube.com/watch?v=YQeVAPYC8YU"
+    ]
+  },
+  "physical": {
+    "dimensions": [150, 430, 150],
+    "weight": 2.8,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [0, 21]
+    }
+  },
+  "wheels": {
+    "Reserved": {
+      "slots": [
+        {
+          "type": "Open"
+        },
+        {
+          "type": "Gobo"
+        },
+        {
+          "type": "Gobo"
+        },
+        {
+          "type": "Gobo"
+        },
+        {
+          "type": "Gobo"
+        }
+      ]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Pan",
+        "angle": "21deg"
```

```suggestion
        "angleStart": "0%"
        "angleEnd": "100%"
```

I can't find any defined angle range.